components/browser/engine-gecko/src/main/java/mozilla/components/browser/engine/gecko/NestedGeckoView.kt

```diff
@@ -116,6 +117,7 @@ open class NestedGeckoView(context: Context) : GeckoView(context), NestedScrolli
         return super.onTouchEvent(event)
     }
 
+    @SuppressLint("WrongThread") // Lint complains startNestedScroll() needs to be called on the main thread
```

Comment:
eek. Let's file a follow-up for that one or is it wrong? Are we actually calling this on the wrong thread?

Reply:
Afaik we are calling this on the right thread. But let me CC @Mugurell who knows best here :)